types/coin_regex.rl

```diff
@@ -0,0 +1,39 @@
+// Code generated by regel using `ragel -Z coin_regex.rl`.
```

Comment:
Comment is misleading, this code wasn't generated right? It was written and then `coin_regex.go` was generated from it?

Reply:
The comment here goes directly to the generated code( coin_regex.go ). Changed in the most recent commit to be more clear!